frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -302,78 +125,145 @@ function Configure() {
 
   useWizardSectionChangeLog()
 
+  const handleSubmit = useCallback(() => {
+    wizardHandleCreate(() => clearWizardState(clearState, false), navigate)
+  }, [wizardHandleCreate, clearWizardState, navigate])
+
+  const handleRefresh = () => {
+    setRefreshing(true)
+    let region = getState(['wizard', 'region'])
+    let chosenRegion = region === 'Default' ? null : region
+    LoadAwsConfig(chosenRegion, () => setRefreshing(false))
+  }
+
+  const navigateWizard = useWizardNavigation(validate)
+
+  const handleOnNavigate: NonCancelableEventHandler<WizardProps.NavigateDetail> =
+    useCallback(
+      ({detail}) => {
+        navigateWizard(detail.reason, detail.requestedStepIndex)
+      },
+      [navigateWizard],
+    )
+
+  const showSecondaryActions = useCallback(() => {
+    return (
+      <SpaceBetween direction="horizontal" size="xs">
+        {currentPage !== 'source' && currentPage !== 'create' && (
+          <Button
+            loading={refreshing}
+            onClick={handleRefresh}
+            iconName={'refresh'}
+          >
+            {t('wizard.actions.refreshConfig')}
+          </Button>
+        )}
+        {editing &&
+          (fleetStatus === 'RUNNING' || fleetStatus === 'STOP_REQUESTED') && (
+            <Button
+              className="action"
+              variant="normal"
+              loading={fleetStatus === 'STOP_REQUESTED'}
+              onClick={stopComputeFleet}
+            >
+              {t('wizard.actions.stopComputeFleet')}
+            </Button>
+          )}
+        {currentPage === 'create' && (
+          <Button onClick={wizardHandleDryRun}>
+            {t('wizard.actions.dryRun')}
+          </Button>
+        )}
+      </SpaceBetween>
+    )
+  }, [editing, fleetStatus, handleRefresh])
+
+  const isLoadingNextStep = useCallback(() => {
+    const aws = getState(['aws'])
+    return !aws
+  }, [getState])
```

Comment:
issue: is this supposed to be updated when the state changes? If so, use `useState` instead of `getState`

Also FYI, there is no need to use `useCallback` here (nor `useMemo`) since you are only interested in the return value, and it's a boolean value (therefore no need to memoize at all)

Reply:
Going to remove entirely the method in favour of `refreshing` value.